src/hm-admin/hm-admin.component.ts

```diff
@@ -1,22 +1,31 @@
 import {Component, Input, OnInit} from '@angular/core';
+import {HTTP_PROVIDERS} from '@angular/http';
 import {ConfigService} from './services/config/config.service';
+import {HydraCore} from './services/hydra/HydraCore.service';
```

Comment:
don't mention Hydra. in the future the one of the goal is to add full support to Hypermedia API